src/Sylius/Bundle/AdminBundle/Resources/config/routing.yml

```diff
@@ -72,3 +72,6 @@ sylius_admin_zone:
 sylius_admin_partial:
     resource: "@SyliusAdminBundle/Resources/config/routing/partial.yml"
     prefix: /_partial
+
+sylius_admin_api_taxon:
+    resource: "@SyliusAdminBundle/Resources/config/routing/api/taxon.yml"
```

Comment:
How about specifying `prefix: /api` here instead of adding it in `api/taxon.yml`?